tests/dragonfly/replication_test.py

```diff
@@ -1294,7 +1294,7 @@ async def info_replication():
     # Give the seeder a bit of time.
     await asyncio.sleep(3)
     logging.debug("running repltakover")
-    await c_replica.execute_command(f"REPLTAKEOVER 10 SAVE")
+    await c_replica.execute_command(f"REPLTAKEOVER 30 SAVE")
```

Comment:
So were not robust enough? :face_exhaling:

Reply:
It looks that with latest changes in replication and because of this 4->2 we don't takeover in time. I have run several tests on (https://github.com/dragonflydb/dragonfly/pull/5392) with this increase and it seems that it is successful on all runs.